package.json

```diff
@@ -15,12 +15,13 @@
   },
   "scripts": {
     "prettier": "prettier --write .",
-    "test": "echo \"Error: no test specified\" && exit 0",
     "build": "rollup -c",
     "release": "npm run build && np",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "deploy-storybook": "storybook-to-ghpages"
+    "deploy-storybook": "storybook-to-ghpages",
+    "test": "jest --watch",
+    "test:ci": "jest --ci --silent"
```

Comment:
With this change, we will need to update the `workflows` to use this `test:ci` script, otherwise, the CI will try to run the tests in watch mode and will fail. As I already have the changes on my side, I'm going to push one commit here to add this change.